src/radical/pilot/agent/staging_output/default.py

```diff
@@ -139,7 +139,7 @@ def work(self, cu):
                         directive, cu['_id'], workdir)
 
                 # Convert the target_url into a SAGA Url object
-                target_url = rs.Url(directive['target'])
+                target_url = ru.Url(directive['target'])
```

Comment:
Let me fix the comment.  In other places that makes sense (for stupid reasons) -- here it does not, as the instance is not forwarded to a saga call.